token-metadata/program/src/constants.rs

```diff
@@ -0,0 +1,69 @@
+/// prefix used for PDAs to avoid certain collision attacks (<https://en.wikipedia.org/wiki/Collision_attack#Chosen-prefix_collision_attack>)
+pub const PREFIX: &str = "metadata";
+
+/// Used in seeds to make Edition model pda address
+pub const EDITION: &str = "edition";
+
+pub const RESERVATION: &str = "reservation";
+
+pub const USER: &str = "user";
+
+pub const BURN: &str = "burn";
+
+pub const COLLECTION_AUTHORITY: &str = "collection_authority";
+
+pub const MAX_NAME_LENGTH: usize = 32;
+
+pub const MAX_SYMBOL_LENGTH: usize = 10;
+
+pub const MAX_URI_LENGTH: usize = 200;
+
+pub const MAX_METADATA_LEN: usize = 1 //key 
```

Comment:
The detailed breakdown of what each literal value is very useful; would be great if we can do the same for the other similarly defined constants.